Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -761,6 +762,22 @@ def send_request(self, path: str, method: str = 'GET', body: dict = None, params
 
         return json_res
 
+    def get_content_type(self, file_name):
+        """Get the correct content type for the POST request.
+
+        Args:
+            file_name: file name
+
+        Returns:
+            the content type - image with right type for images , and general for other types..
+        """
+        file_type = None
+        if not file_name:
+            demisto.log("file name was not supllied, uploading with general type")
+        else:
+            file_type, _ = mimetypes.guess_type(file_name)
+        return file_type or '*/*'
```

Comment:
```suggestion
        if not file_name:
            demisto.log("file name was not supllied, uploading with general type")
            return '*/*'
        else:
            file_type, _ = mimetypes.guess_type(file_name)
            return file_type
```
Seems clearer to me

Reply:
`mimetypes.guess_type` can return `None` if the type was not identified. 
Any better way to check for this?